### PR TITLE
[v7r0] Some fixes for running pilots

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*WMS
FIX: SiteDirector: Pilot3 option easier to interpret

*Core
NEW: dirac-install accepts userEnvVariables switch, adds user-requested env variables to *rc files

ENDRELEASENOTES

### CODE REVIEW

File: Core/scripts/dirac-install.py [hunk 1]
```diff
@@ -1678,7 +1679,9 @@ def discoverModules(modules):
            ('x:', 'external=', 'external version'),
            ('  ', 'createLink', 'create version symbolic link from the versions directory. This is equivalent to the \
            following command: ln -s /opt/dirac/versions/vArBpC vArBpC'),
-           ('  ', 'scriptSymlink', 'Symlink the scripts instead of creating wrapper')
+           ('  ', 'scriptSymlink', 'Symlink the scripts instead of creating wrapper'),
+           ('  ', 'userEnvVariables',
```
Comment (arrabito) [inline Core/scripts/dirac-install.py]: It seems that to work correctly you should replace with:

('  ', 'userEnvVariables=',

Comment (fstagni) [reply Core/scripts/dirac-install.py]: Correct. Done.

**Comment by arrabito** [comment]
When I try the command:

python dirac-install.py -V "defaults-CTA.cfg" -e "CTA" -i '27' -l 'CTA' -r "v1r55p1" --userEnvVariables DIRACSYSCONFIG:::pilot.cfg

I also get the following error:

2019-12-19 12:22:01 UTC dirac-install [NOTICE]  Running externals post install...
2019-12-19 12:22:01 UTC dirac-install [NOTICE]  Creating /sps/hep/cta/lupm/arrabito/test_dirac_v7/test_pilot/bashrc
2019-12-19 12:22:01 UTC dirac-install [ERROR] too many values to unpack

**Comment by fstagni** [comment]
try with `--userEnvVariables=DIRACSYSCONFIG:::pilot.cfg`

**Comment by arrabito** [comment]
I get the same error.

**Comment by fstagni** [comment]
Right. Try now.

**Comment by arrabito** [comment]
It works fine now.

**Comment by fstagni** [comment]
Please review

**Pull request merged**